src/main/scala/io/citrine/lolo/bags/Bagger.scala

```diff
@@ -15,14 +15,19 @@ import scala.reflect._
   * Created by maxhutch on 11/14/16.
   *
   * @param method  learner to train each model in the ensemble
-  * @param numBags number of models in the ensemble
-  */
+  * @param numBags number of base models to aggregate (default of -1 sets the number of models to the number of training rows)
+  * @param useJackknife whether to enable jackknife uncertainty estimate
+  * @param biasLearner learner to use for estimating bias
+  * @param uncertaintyCalibration whether to enable empirical uncertainty calibration
+  * @param disableBootstrap whether to disable bootstrap (useful when `method` implements its own randomization)
+  * */
 case class Bagger(
                    method: Learner,
                    numBags: Int = -1,
                    useJackknife: Boolean = true,
                    biasLearner: Option[Learner] = None,
                    uncertaintyCalibration: Boolean = false,
+                   disableBootstrap: Boolean = false,
```

Comment:
I assume you added this because the standard behavior for ExtraRandomTrees is to not bootstrap.  However, this setting isn't actually used in the ExtraRandomTrees learner.